circle.yml

```diff
@@ -144,19 +144,10 @@ defaults:
     run:
       name: "Test shared Hera (wabt)"
       command: |
+        SO=$([ $(uname) = Darwin ] && echo dylib || echo so)
```

Comment:
I think `test-wabt` should also be added to the list of tasks run on MacOS, to make full use of these lines.